### ISSUE TITLE
Geodata importer

### ISSUE THREAD

**Issue by v1r0x** [eScienceCenter/Spacialist#197]
We need an importer to import different kinds of geodata. Please add items to the list if I missed something.
- [x] Import geo objects (different reference systems, see #191 )
  - [x] WKT/WKB
  - [x] xy(z) (csv, excel)
  - [x] Shape-Files (.shp); see: [Shp2pgsql](http://www.geoportal.rlp.de/mediawiki/index.php/Shp2pgsql)
  - [x] KML/KMZ with `ST_GeomFromKML`
  - [x] ~~SpatiaLite; see: [ogr2ogr -f SQLite](https://trac.osgeo.org/postgis/wiki/SpatiaLite)~~

**Comment by derntl** [comment]
For each project we need to specify a spatial reference to be used for original geometry data. So this needs to be part of project configuration. For the map we just transform to geography/4326.

**Comment by v1r0x** [comment]
This should be a part of the GIS-View of the ToDo list in #198

**Comment by v1r0x** [comment]
I'm currently working on the importer as part of #246 .
I started with csv, which is easy to parse. For the other formats (WKT, shp, KML/KMZ) there are parsers which only support geojson as output format. I found a parser doing the same for csv, but it only supports lat/lon and no altitude attribute. After some research it looks like all of the parsers are lat/lon only.
Question is: Is altitude important for us? @dirkseidensticker @AngusLeBoeuf
